app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,6 +68,182 @@ angular.module('confRegistrationWebApp')
       return _.find($scope.blocks, {id: blockId});
     };
 
+    var setRegistrationAsCompleted = function() {
+      registration = angular.copy(registration);
+      registration.completed = true;
+
+      RegistrationCache.update('registrations/' + registration.id, registration, function () {
+        RegistrationCache.emptyCache();
+        if(conference.registrationCompleteRedirect){
+          $window.location.href = conference.registrationCompleteRedirect;
+        }else{
+          $route.reload();
+        }
+      }, function (data) {
+        $scope.currentRegistration.completed = false;
+        $scope.submittingRegistration = false;
+        modalMessage.error(data.error ? data.error.message : 'An error occurred while submitting your registration.');
+      });
+    };
+
+    ////// START EVENT-433 //////
+
+    if (!$rootScope.includeSpouseRegistrationClicked) {
+      // Get spouse registration
+      // End point: getSpouseRegistration = /conferences/conferenceId/registrations/spouse
+      ///// TEST DATA end point url: /conferences/conferenceId/registrations/current /////
+      $http.get('conferences/' + conference.id + '/registrations/current')
+        .then(function(response) {
+          if (response.status === 200) {
+            $scope.spouseRegistration = response.data;
+
+            ///// Test Data /////
+            var id = uuid();
+            $scope.spouseRegistration.registrants[0].id = id;
+            $scope.spouseRegistration.registrants[0].registrationId = uuid();
+            $scope.spouseRegistration.registrants[0].userId = uuid();
+            $scope.spouseRegistration.registrants[0].email = 'janeworner3@hotmail.com';
+            $scope.spouseRegistration.registrants[0].firstName = 'Jane';
+            $scope.spouseRegistration.registrants[0].lastName = 'Worner3';
+            $scope.spouseRegistration.registrants[0].answers[1] = {id: uuid(), registrantId: id, blockId: 'b56c9fb7-9643-47d9-9408-ead1bcd6ab31', value: 'janeworner3@hotmail.com'};
```

Comment:
I just matched them to the same indices to the current registrant answers.  Are you seeing something different?